components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -232,7 +232,26 @@ public void handle(HttpServletRequest request, HttpServletResponse response) thr
                         log.debug("Session data key  :  " + key);
                     }
                 }
-                log.error("Context does not exist. Probably due to invalidated cache");
+
+                String userAgent = request.getHeader("User-Agent");
+                String referer = request.getHeader("Referer");
+                String parameters = "[";
+                Map params = request.getParameterMap();
+                for (Object o : params.keySet()) {
+                    String key = (String) o;
+                    String value = ((String[]) params.get(key))[0];
+                    if ("password".equalsIgnoreCase(key) && !value.isEmpty()) {
+                        continue;
```

Comment:
I don't think that password is the only sensitive information which could arrive here. Basically there might be number of sensitive things which might be coming to this place based on the type of authenticator they used.

Reply:
So shall I remove the parameter list then?